src/main/java/times/DukeTime.java

```diff
@@ -0,0 +1,76 @@
+package times;
+
+import exceptions.DukeException;
+import exceptions.WrongTimeFormatDukeException;
+
+import java.time.LocalDateTime;
+import java.time.Month;
+
+public class DukeTime {
+    private LocalDateTime time;
+    private static final int DEFAULT_YEAR = 2022;
+    private static final Month DEFAULT_MONTH = Month.of(1);
+    private static final int DEFAULT_DAY = 1;
+    private static final int DEFAULT_HOUR = 23;
+    private static final int DEFAULT_MIN = 59;
+    public DukeTime(String timeString) throws WrongTimeFormatDukeException {
+        String[] sequence = timeString.split(" ");
+        try{
+            int[] dateIntElements;
+            switch (sequence.length) {
+            case 1:
```

Comment:
Avoid using magic literals such as `1`, consider extracting this into a constant.